scipy/sparse/linalg/dsolve/_superlumodule.c

```diff
@@ -271,27 +271,34 @@ static char gstrf_doc[] = "gstrf(A, ...)\n\
 performs a factorization of the sparse matrix A=*(N,nnz,nzvals,rowind,colptr) and \n\
 returns a factored_lu object.\n\
 \n\
-arguments\n\
----------\n\
+Paramerters\n\
+-----------\n\
 \n\
 Matrix to be factorized is represented as N,nnz,nzvals,rowind,colptr\n\
   as separate arguments.  This is compressed sparse column representation.\n\
 \n\
-N         number of rows and columns \n\
-nnz       number of non-zero elements\n\
-nzvals    non-zero values \n\
-rowind    row-index for this column (same size as nzvals)\n\
-colptr    index into rowind for first non-zero value in this column\n\
-          size is (N+1).  Last value should be nnz. \n\
+N : int \n\
+    number of rows and columns\n\
+nnz : int\n\
+    number of non-zero elements\n\
+nzvals\n\
+    non-zero values \n\
```

Comment:
The CI/doc build seems happy. I don't really have an opinion on formatting C lang docs this way, but some of the parameters were given types and others were not?

Reply:
Yeah, I don't particularly use this function, so I only put what was obvious.